packages/e2e-test-utils/src/mocks/set-up-response-mocking.js

```diff
@@ -10,10 +10,13 @@ let requestMocks = [];
 
 /**
  * Sets up mock checks and responses. Accepts a list of mock settings with the following properties:
- *   - match: function to check if a request should be mocked.
- *   - onRequestMatch: async function to respond to the request.
  *
- * Example:
+ * - `match`: function to check if a request should be mocked.
+ * - `onRequestMatch`: async function to respond to the request.
+ *
+ * @example
+ *
+ *```js
```

Comment:
If every other line in the JSDoc is preceded by a `*` **_and a space_**, should the space not also be present here?

Reply:
Good catch. Adjusted the spacing.